parsers/ocaml.c

```diff
@@ -594,13 +629,15 @@ static const char *contextDescription (contextType t)
 	case ContextValue:
 		return "value";
 	case ContextModule:
-		return "Module";
+		return "module";
```

Comment:
Now I understand the need of this change.
This line is needed to make the kind name used in scope information and the kind name in OcamlKinds struct, which is used in --list-kinds, the same.

You updated expected.tags. It is correct.